Mathlib/Algebra/Hom/Centroid.lean

```diff
@@ -112,6 +111,7 @@ instance : CoeFun (CentroidHom α) fun _ ↦ α → α :=
 /- Porting note:
 `theorem to_fun_eq_coe {f : CentroidHom α} : f.toFun = (f : α → α) := rfl`
 removed because it is now a tautology -/
+#noalign centroid_hom.to_fun_eq_coe
```

Comment:
Is this still true? Don't we use `FunLike.coe` on the RHS?

Reply:
Nice catch, thanks